ostree/ostree_src.go

```diff
@@ -34,14 +36,15 @@ import (
 import "C"
 
 type ostreeImageSource struct {
-	ref    ostreeReference
-	tmpDir string
-	repo   *C.struct_OstreeRepo
+	ref        ostreeReference
+	tmpDir     string
+	repo       *C.struct_OstreeRepo
+	compressed map[digest.Digest]digest.Digest
```

Comment:
Please at least add a comment here saying what is the key and what is the value of the map. (Ideally the field name would be self-explanatory, but that seems difficult.)